src/lib.rs

```diff
@@ -91,6 +92,7 @@ decl_storage! {
         // enclave index of the worker that recently committed an update
         pub WorkerForShard get(fn worker_for_shard) : map hasher(blake2_128_concat) ShardIdentifier => u64;
         pub ConfirmedCalls get(fn confirmed_calls): map hasher(blake2_128_concat) H256 => u64;
+        //pub ConfirmedBlocks get(fn confirmed_blocks): map hasher(blake2_128_concat) H256 => u64;
```

Comment:
IMO, it would be better to delete this instead of leaving it commented

Reply:
we'll clean this up